engines/xeen/font.cpp

```diff
@@ -311,7 +311,8 @@ void FontSurface::writeCharacter(char c, const Common::Rect &clipRect) {
 }
 
 char FontSurface::getNextChar() {
-	if (Common::RU_RUS == lang) return *_displayString++;
+	if (Common::RU_RUS == Common::parseLanguage(ConfMan.get("language")))
```

Comment:
Isn't there already a lang variable in the file from the earlier commit. If so, it should be used, since it'll be a lot more performant to simply check a variable for each character than to re-parse a language string and convert it to a number.

Reply:
Fixed in later commits